### PR TITLE
Modify Clear functionality and Sort functionality in UG

### LINKED ISSUE #138: Update on UG v2 Amanda

### PR DESCRIPTION
**Pull request by github-amanda**
1. Add clear by location functionality
2. Add sort by ascending or descending order functionality

Closes #138

### CODE REVIEW

**Review by JayChenYJ — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -310,24 +310,38 @@ Examples:
...
 
-  * `quantity` is case-insensitive. Keying in `sort QUANTITY` is also a valid command.
+  * `quantity` is case-insensitive. Keying in `sort QUANTITY asc` is also a valid command.
     <br></br>
 
   </div>
   
   Example:
-  * `sort quantity` sorts the items in the displayed list in ascending order of quantity.
+  * `sort quantity acs` sorts the items in the displayed list in ascending order of quantity.
```
Comment (Md-Fazil) [inline docs/UserGuide.md:327]: I believe this is a typo. Shouldn't it be "sort quantity asc"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -310,24 +310,38 @@ Examples:
...
-You can use this command in 2 different ways.
+You can use this command in 3 different ways.
 
-* #### 3.8.1. Sorting items by quantity
-  Format 1: `sort quantity`
+* #### 3.8.1. Sorting items by ascending quantity
+  Format 1: `sort quantity asc`
   <div markdown="block" class="alert alert-info">
 
   **:information_source: Note about the sort by expiry date command:**<br>
```
Comment (Md-Fazil) [inline docs/UserGuide.md:319]: I think this should be changed to "Note about the sort by quantity command".

File: docs/UserGuide.md [hunk 3]
```diff
@@ -310,24 +310,38 @@ Examples:
...
   
   Example:
-  * `sort quantity` sorts the items in the displayed list in ascending order of quantity.
+  * `sort quantity acs` sorts the items in the displayed list in ascending order of quantity.
+
+* #### 3.8.2. Sorting items by descending quantity
+  Format 2: `sort quantity desc`
+  <div markdown="block" class="alert alert-info">
+
+  **:information_source: Note about the sort by expiry date command:**<br>
```
Comment (Md-Fazil) [inline docs/UserGuide.md:333]: As mentioned, this should be changed as well.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -345,7 +359,27 @@ You can use this command in 2 different ways.
...
+
+* #### 3.9.1. Clear all items in entire inventory
+  Format 1: `clear`
+  <div markdown="block" class="alert alert-info">
+
+* #### 3.9.2. Clear all items in a specific location
+  Format 2: `clear l/LOCATION`
+  <div markdown="block" class="alert alert-info">
+
+  **:information_source: Notes about the command:**<br>
```
Comment (Md-Fazil) [inline docs/UserGuide.md:372]: Would it be better if we state "notes about the clear command"?

**Review by Md-Fazil — COMMENTED** [review]
LGTM!

**Pull request merged**